src/logic/map_objects/tribes/worker.cc

```diff
@@ -533,7 +533,7 @@ struct FindNodeSpace {
 		for (uint8_t dir = FIRST_DIRECTION; dir <= LAST_DIRECTION; ++dir) {
 			FCoords const neighb = egbase.map().get_neighbour(coords, dir);
 
-			if (!(neighb.field->nodecaps() & MOVECAPS_WALK) &&
+			if (!(neighb.field->maxcaps() & MOVECAPS_WALK) &&
```

Comment:
You need to get rid of `ignoreimmovable` now, the change makes it obsolete and it might only cause corner-case bugs.